config/auditable.php

```diff
@@ -10,7 +10,7 @@
      * Route middleware for auditable endpoints.
      */
     'route_middleware' => [
-        'api',
+        'auth:api',
```

Comment:
The `api` middleware does not prevent unauthenticated access.

Thanks @fernando-aloware for catching this bug!